src/container.jl

```diff
@@ -124,7 +124,8 @@ end
 
 # https://github.com/aws/amazon-ecs-agent/issues/1119
 # Note: For AWS Batch array jobs the "ecs/<job_id>" does not include the array index
-const CGROUP_REGEX = r"\/(?:docker|ecs\/[0-9a-f]{32})\/(?<container_id>[0-9a-f]{64})\b"
+# Note: GitHub Actions uses "actions_job"
+const CGROUP_REGEX = r"\/(?:docker|ecs\/[0-9a-f]{32}|actions_job)\/(?<container_id>[0-9a-f]{64})\b"
```

Comment:
Could we leave a comment explaining the regex, and an example of what's to match?

Reply:
I'll add an explanation here